### PR TITLE
Remove c3.medium from kexec bypass

### PR DESCRIPTION
**Pull request by dustinmiller**
We opted for BIOS over UEFI for this plan, so we can kexec in now

- [x] Changelog updated

### CODE REVIEW

File: docker/scripts/osie.sh [hunk 1]
```diff
@@ -425,7 +425,7 @@ EOF_tty
 	# Backup phone-home
 
 	# Add phone-home to OS
-	nonsystemdoses=(ubuntu_14_04 ubuntu_14_04_image scientific_6)
+	nonsystemdoses=(_14_04 ubuntu_14_04_image scientific_6)
```
Comment (ScottGarman) [inline docker/scripts/osie.sh]: To my untrained eye, I'm not clear on why this change was made - can you explain? Doesn't need a code comment, I'm just looking to learn.

Comment (dustinmiller) [reply docker/scripts/osie.sh]: This was unintentional, great catch

**Comment by ScottGarman** [comment]
:+1:

**Review by mmlb — APPROVED** [review]
